security/tpm/src/AuthenticationProviderTpmHsm.cs

```diff
@@ -3,10 +3,9 @@
 
 using System;
 using System.Runtime.InteropServices;
-using Microsoft.Azure.Devices.Authentication;
 using Tpm2Lib;
 
-namespace Microsoft.Azure.Devices.Provisioning.Security
+namespace Microsoft.Azure.Devices.Authentication
 {
     /// <summary>
```

Comment:
It looks like this file wasn't moved to the Authentication directory like the others

Reply:
It was moved but I did not delete a copy of it from the TPM directory.

Updated it in recent change.